helics/capi.py

```diff
@@ -1392,11 +1392,13 @@ def message_id(self, v):
 
 
 class HelicsQuery(_HelicsCHandle):
-    pass
+    def __init__(self, handle, cleanup=False):
+        super(HelicsQuery, self).__init__(handle, cleanup)
```

Comment:
Having `pass` should be equivalent to these lines, so I'm not quite sure why this change is needed.

Reply:
When we just had `pass` in there we were getting the error below.
```
    f(buffer.handle, string, len(string), err)
      ^^^^^^^^^^^^^
AttributeError: cdata 'void *' has no attribute 'handle'
```

Calling the super's init() solved the problem. Generally, I didn't think a class called it's super's init but you know WAY MORE about these things than I do.

Reply:
If you're interested @kdheepak , I'm sure we can set up a call to go over this. Or, you can try running the [connector example](https://github.com/GMLC-TDC/HELICS-Examples/tree/main/user_guide_examples/advanced/advanced_connector/interface_creation) with and without the above changes to capi.py to see if you get similar behavior. You'll need to build the HELICS "dev" branch to get the connector app.

Reply:
Does Github update automatically or manually? Right now it's showing v3.4.0 as the latest release.
![Screenshot 2024-02-09 at 9 59 01 AM](https://github.com/GMLC-TDC/pyhelics/assets/8697189/c1844162-16fd-449e-b236-e6048d29adad)

Reply:
apparently have to actively select a check box for that now,  now 3.5 should be the latest

Reply:
I'm still seeing the checkbox selected by default when I go to draft a release. It might remember the last selection that was made for a release.

I figured it just hadn't been marked as latest while the remaining interfaces were being updated.

Reply:
This error is telling me that `buffer` is a `cdata 'void *'` object and has no attribute 'handle'.

```
    f(buffer.handle, string, len(string), err)
      ^^^^^^^^^^^^^
AttributeError: cdata 'void *' has no attribute 'handle'
```

That's being called in:

https://github.com/GMLC-TDC/pyhelics/blob/5516142dfa81de726357c6986444c5f310240e24/helics/capi.py#L9595-L9609

You can see that `buffer` should be a `HelicsQueryBuffer` and not a `cdata 'void *'`. That's telling me that the wrong data is being passed in for some reason.

I do see the `HelicsQueryBuffer` is being created correctly here:

https://github.com/GMLC-TDC/HELICS-Examples/blob/main/user_guide_examples/advanced/advanced_connector/interface_creation/Charger.py#L169
https://github.com/GMLC-TDC/HELICS-Examples/blob/main/user_guide_examples/advanced/advanced_connector/interface_creation/Battery.py#L116

I'll try to run the example to see what is going on when I get the chance.